src/stapi_fastapi/routers/product_router.py

```diff
@@ -105,8 +113,14 @@ async def get_product_constraints(self: Self) -> JsonSchemaModel:
         """
         return self.product.constraints
 
+    async def get_product_order_parameters(self: Self) -> JsonSchemaModel:
```

Comment:
This doesn't need to be `async`. The fact that `get_product_constraints` is because removing the `async` was missed, neither needs to be `async`.

Reply:
resolved by tyler